input.lua

```diff
@@ -1,9 +1,14 @@
 function courseplay:onMouseEvent(posX, posY, isDown, isUp, mouseButton)
 	--RIGHT CLICK
 	-- Input binding debug
-	local vehicle = g_currentMission.controlledVehicle		
+	local vehicle = g_currentMission.controlledVehicle			
 	if not vehicle or not vehicle.hasCourseplaySpec then return end
-  courseEditor:updateMouseState(vehicle, posX, posY, isDown, isUp, mouseButton)
+
+	--if the vehicle is attached to another vehicle, disable mouse input 
```

Comment:
and here, a fourth time. We may already have something for this, and probably name it more generic, like isCourseplayEnabled?